packages/tests/tsconfig.json

```diff
@@ -1,10 +1,53 @@
 {
-  "extends": "../../tsconfig",
   "compilerOptions": {
+    "incremental": true,
+    "target": "es2020",
```

Comment:
why are we not extending from the original root `tsconfig` here?

Reply:
Well spotted, not sure how I ended up reverting the change.